spec/index.bs

```diff
@@ -1604,7 +1604,11 @@ dictionary GPUImageBitmapCopyView {
 <script type=idl>
 interface mixin GPUProgrammablePassEncoder {
     void setBindGroup(unsigned long index, GPUBindGroup bindGroup,
-                      optional sequence<GPUBufferSize> dynamicOffsets = []);
+                      optional sequence<unsigned long> dynamicOffsets = []);
+    void setBindGroup(unsigned long index, GPUBindGroup bindGroup,
+                      Uint32Array dynamicOffsetsData,
+                      optional unsigned long dynamicOffsetsDataStart = 0,
```

Comment:
I just realized this may need to be longer than 32 bits, because it's an offset into a Uint32Array which could theoretically be more than 16GiB.

Reply:
good point! it's not a problem to use u64 here

Reply:
Done